docs/src/SUMMARY.md

```diff
@@ -0,0 +1,15 @@
+# Documentation
+
+* [Introduction](./README.md)
+* [Effects](./configuration/effects.md)
+* [Loots](./configuration/loots.md)
+* [Projectiles](./configuration/projectiles.md)
+* [Skills](./configuration/skills.md)
+* [Characters](./configuration/characters.md)
+* [Game](./configuration/game.md)
+* [Player attributes](./attributes/players.md)
+* [Projectile attributes](./attributes/projectiles.md)
+* [Architecture](./architecture/backend_architecture.md)
+* [Metrics and monitoring](./metrics_and_monitoring/metrics-and-monitoring.md)
+* [Load Testing](./testing/load_testing/load_testing.md)
+* [Simulating High Latency](./testing/simulating_latency.md)
```

Comment:
Can we split this sections into titles? for example:
Configuration
- Introduction
- Effects
- Loots
- Projectiles
- Skills
- Characters
- Game

Reply:
Looking good!